crates/testing/src/test_builder.rs

```diff
@@ -152,14 +152,14 @@ impl TestMetadata {
             completion_task_description: CompletionTaskDescription::TimeBasedCompletionTaskBuilder(
                 TimeBasedCompletionTaskDescription {
                     // Increase the duration to get the expected number of successful views.
-                    duration: Duration::new(340, 0),
+                    duration: Duration::new(3400, 0),
```

Comment:
I think we probably should keep these as is.  If the test is taking more than 5 minutes it's probably some kind of deadlock anyway.